conf/base.config

```diff
@@ -7,6 +7,16 @@
     the PATH. Runs in `local` mode - all jobs will be run on the logged in environment.
 ----------------------------------------------------------------------------------------
 */
+// Set default parameters that may be overwritten
+params {
+    download_annots = 'none'
+    min_num_spp_per_og = '4'
+    min_num_grp_per_og = '1'
+    max_copy_num_spp_tree = '5'
+    max_copy_num_gene_trees = '10'
+    tree_model = 'TEST'
```

Comment:
should the default be something else?

Reply:
What do you mean? 

As is, I've actually tried to specify some sensible defaults - I don't actually intend to use these myself, but they are are least less sensitive to specific details of any one dataset.

Reply:
What I mean is: if someone ran this with the default value of "TEST", will the pipeline fail or run in a semi-intelligent way? If not, we should either have a sensible default or not have a default and throw an error if not specified.

Reply:
Yeah, this is actually the most foolproof. Basically it makes it so that it's agnostic to whether input sequences are amino acids or nucleotides. I suppose since I haven't tested the workflow with nucleotides, I could just set a sensible model for AA sequences?